### PR TITLE
Issue #27: Unfollow

### PR DESCRIPTION
**Pull request by cottonswab**
- Implemented back end for unfollow feature (server>following.js)
- Implemented front end for unfollow feature (client>following.js)
- Merging changes to dependencies
- Updated install packages

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\client",
```
Comment (ginywiny) [inline .vs/VSWorkspaceState.json:4]: Is it normal for these paths to take two backslashes "\\"?

File: server/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bcrypt": "^4.0.0",
```
Comment (ginywiny) [inline server/package.json:15]: Be careful when pushing new package.json updates, as they can cause some build issues. Although it seems fine here, just mentioning for future notice.

**Review by ginywiny — APPROVED** [review]

File: server/package.json (hunk 2 above)
Comment (cottonswab) [reply server/package.json:15]: Duly noted.

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (cottonswab) [reply .vs/VSWorkspaceState.json:4]: yes, that is part of the syntax

**Review by ma-brandonjh — APPROVED** [review]
"Unfollow" feature works. Hopefully, we can get TravisCI to work again.

**Pull request merged**